core/arch/arm/plat-vexpress/main.c

```diff
@@ -114,14 +114,14 @@ static void read_console(void)
 	}
 }
 
-static enum itr_return console_itr_cb(struct itr_handler *hdl)
+static enum itr_return console_itr_cb(struct itr_handler *hdl __unused)
 {
 	if (notif_async_is_started()) {
 		/*
 		 * Asynchronous notifications are enabled, lets read from
 		 * uart in the bottom half instead.
 		 */
-		interrupt_disable(hdl->chip, hdl->it);
+		console_data.chip.ops->rx_intr_disable(&console_data.chip);
```

Comment:
Having two abbreviations for "interrupt" (itr and intr) is somewhat unfortunate but I suppose it's OK for now.